README.md

```diff
@@ -93,6 +93,32 @@ overwrite the pre-processed data.
   [Reproducible Research](https://the-turing-way.netlify.app/reproducible-research/renv.html)
   from [The Turing Way](https://the-turing-way.netlify.app/welcome.html).
 
+## Tests
+
+This repo finally has some tests! wahoo! Unfortunately, there are memory issues
+getting in the way of letting us run all of the tests via github actions
+([PUDL issue #1457](https://github.com/catalyst-cooperative/pudl/issues/1457)).
+
+### Regenerate All Outputs & Validate
+
+``sh
+pytest test/integration/rmi_out_test.py
```

Comment:
You need to enclose these code blocks with triple-backticks, not double to get them set apart and syntax highlighted as `sh`

Reply:
When you run `tox` all by itself, it will run whichever test environment is set as the default at the top of `tox.ini`, in this case the `ci` testenv: 

```ini
[tox]
envlist = ci
```

So in this case, yes -- just running `tox` with no arguments is the way to go.  You can also look `.github/workflows/tox-pytest.yml` to see how we're running it there:

```yaml
    - name: Run pytest using Tox
      env:
        API_KEY_EIA: ${{ secrets.API_KEY_EIA }}
      run: |
        conda run -n pudl-rmi tox
``` 

Aside: I notice that the tests are still using the PUDL DB that we published in November. Is that correct?

```yaml
    - name: Cache PUDL DB
      uses: actions/cache@v3.0.2
      with:
        path: ~/pudl-work/sqlite/pudl.sqlite
        # The last time we deployed Datasette. Update if we redeploy.
        key: "2021-11-12"

    - name: Download PUDL DB and log pre-test PUDL workspace contents
      run: |
        mkdir -p ~/pudl-work/sqlite/
        if [ ! -f ~/pudl-work/sqlite/pudl.sqlite ]
        then curl -o ~/pudl-work/sqlite/pudl.sqlite https://data.catalyst.coop/pudl.db
        fi
        find ~/pudl-work/
```

Reply:
We really gotta fix the memory issues in this repo. 😑

Reply:
@zaneselvans , no i assume the november DB is not correct. What is the most up to date one?? (and how would I check that and update it in the future?)

Reply:
Datasette was last deployed on 2022-05-05.

To check this I logged in to GCP and looked under "Cloud Run" which is the tools we use to deploy Datasette.